### PR TITLE
[now-node] Cleanup debug output

### PR DESCRIPTION
**Pull request by thasophearak**
This PR add missing annotate for traced file.

### CODE REVIEW

File: packages/now-node/src/index.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ async function compile(
   });
 
   debug('traced files:');
-  debug('\t' + fileList.join('\n\t'));
+  console.log('\t' + fileList.join('\n[now-builder-debug] \t'));
```
Comment (styfle) [inline packages/now-node/src/index.ts]: Let's delete these two lines. Its never relevant to the user. These were more for debugging ourselves.

Comment (styfle) [reply packages/now-node/src/index.ts]: Let's also remove the `debug('compiling es module file ' + path);` line too

Comment (styfle) [reply packages/now-node/src/index.ts]: And also remove `debug('compiling typescript file ' + relPath);`

Comment (thasophearak) [reply packages/now-node/src/index.ts]: Removed 🙌

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**